src/WebJobs.Extensions.CosmosDB/WebJobs.Extensions.CosmosDB.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <Version>$(CosmosDBVersion)</Version>
+    <Version>$(CosmosDBVersion)-preview</Version>
```

Comment:
Will this be the only preview? I usually use `preview1` just in case I need another one...

Reply:
Done